gallery/src/components/NxTabs/NxTabsSimpleExample.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState} from 'react';
+
+import {
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel
+} from '@sonatype/react-shared-components';
+
+const NxTableSimpleExample = () => {
+  const [activeTabId, setActiveTabId] = useState('tab-1');
+
+  return (
+    <NxTabs>
```

Comment:
Is there any value in this component ? seems the stateful version is more useful, and if it were me, would just have NxTabs inherit the statelful functionality

Reply:
I can see a lot of cases where you might want more complex behavior than "click-to-show" an individual tab. Maybe you want content within one of the tabs to activate another tab for example. The stateful version is probably useful in the majority of cases, but you may need the stateless version to give yourself more control and that means you have to be able to control the props that mark a tab as active.

Reply:
Also, I can't just expose the `setActiveTabId` functionality, though I suppose I could make the active tab id a prop that we set to default the active tab when the component is first rendered. That said, if you wanted to do this any other way (like maybe not setting an id on the tabas or something) that design would be more restrictive. Generally in the RSC we've been creating a stateless version of the component first, then adding a stateful version that provides behavior around the stateless version as it's more generally useful and doesn't box you into decisions we made in the stateful version.

Reply:
fair enough, thanks for the explanation :)

Reply:
> Is there any value in this component ? seems the stateful version is more useful, and if it were me, would just have NxTabs inherit the statelful functionality

See here for RSC's general philosophy around starting with stateless components and then optionally adding stateful wrappers: https://github.com/sonatype/sonatype-react-shared-components/blame/master/README.md#L51